src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,48 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final String DATE_PATTERN = "dd.MM.yyyy";
+    private static final int WORK_DATE_INDEX = 0;
+    private static final int WORK_HOURS_INDEX = 2;
+    private static final int HOUR_SALARY_INDEX = 3;
+    private static final String DATA_SEPARATOR = " ";
```

Comment:
not a mistake, but you can group constants of the same data type (string) together